src/langcheck/augment/en/_keyboard_typo.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import annotations
+
+from nlpaug.augmenter.char.keyboard import KeyboardAug
+
+
+def keyboard_typo(
+    texts: list[str] | str,
+    **kwargs,
+) -> list[str]:
+    '''Generate keyboard typo perturbed texts for augmentation.
+
+    Args:
+        texts: List of texts to be augmented.
+
+    .. note::
+        Any argument that can be passed to
+        `nlpaug.augmenter.char.keyboard.KeyboardAug
+        <https://nlpaug.readthedocs.io/en/latest/augmenter/char/keyboard.html#nlpaug.augmenter.char.keyboard.KeyboardAug>`_
+        is acceptable. Some of the more useful ones from `nlpaug` document are
+        listed below:
+            - ``aug_char_p`` (float): Percentage of character (per token)
+              will be augmented.
+            - ``aug_char_min`` (int): Minimum number of character will be
+              augmented.
+            - ``aug_char_max`` (int): Maximum number of character will be
+              augmented.
+            - ``aug_word_p`` (float): Percentage of word will be augmented.
+            - ``aug_word_min`` (int): Minimum number of word will be augmented.
+            - ``aug_word_max`` (int): Maximum number of word will be augmented.
```

Comment:
```suggestion

        - ``aug_char_p`` (float): Percentage of character (per token)
          will be augmented.
        - ``aug_char_min`` (int): Minimum number of character will be
          augmented.
        - ``aug_char_max`` (int): Maximum number of character will be
          augmented.
        - ``aug_word_p`` (float): Percentage of word will be augmented.
        - ``aug_word_min`` (int): Minimum number of word will be augmented.
        - ``aug_word_max`` (int): Maximum number of word will be augmented.

```

There are some errors when building the doc. Seemingly unindenting the bullet list & inserting blanks before & after that will resolve the issue.

Reply:
I fixed the sphinx error.